open_schools_platform/ticket_management/tickets/rules.py

```diff
@@ -19,8 +19,7 @@ def ticket_recipient_access():
     @rules.predicate
     @predicate_input_type_check
     def organization_access(user: User, ticket: Query):
-        from open_schools_platform.organization_management.organizations.selectors import get_organizations_by_user
-        return ticket.recipient in get_organizations_by_user(user)
+        return has_related_organization(user, ticket.recipient)
 
```

Comment:
you can delete it and simple write 
return user.has_perm('orgniazations.organization_access', ticket.recipeint)

Reply:
it is better to write like you did it for query with dictionaries